string command
string component_name
string parametr_name
---
bool status